src/catalog/catalog.cpp

```diff
@@ -482,17 +489,18 @@ ResultType Catalog::CreateIndex(const std::string &database_name,
                            " to create index");
 
   // check if table exists
-  auto table_object = database_object->GetTableObject(table_name, schema_name);
+  auto table_object = database_object->GetTableObject(table_name, schema_name, session_namespace);
   if (table_object == nullptr)
     throw CatalogException("Can't find table " + schema_name + "." +
                            table_name + " to create index");
 
   IndexConstraintType index_constraint =
       unique_keys ? IndexConstraintType::UNIQUE : IndexConstraintType::DEFAULT;
 
+  //schema not sure. should get from the table object
```

Comment:
The first character in the comment should be upper-case (consistent with others)